src/pages/BDI_DataImportDeleteBTN.page

```diff
@@ -3,28 +3,36 @@
     tabStyle="DataImport__c"
     showHeader="true"
     sidebar="false"
-    action="{!buttonClick}"
     docType="html-5.0"
     standardStylesheets="true">
 
     <apex:slds />
     <apex:stylesheet value="{!URLFOR($Resource.CumulusStaticResources, '/npsp-slds/npsp-common.css')}" />
 
-    <div class="slds-scope slds-p-bottom_x-large">
-        <apex:form id="idForm" styleClass="slds-p-bottom_x-large">
-    
-        <c:UTIL_PageMessages />
 
-        <div class="slds-card__header slds-grid">
-            <div class="slds-no-flex">
-                <apex:commandButton value="{!$Label.bdiBtnClose}" 
-                    action="{!close}" 
-                    immediate="true" 
-                    styleClass="slds-button slds-button_neutral" />
+        <div class="slds-scope slds-p-bottom_x-large">
+            <apex:form id="idForm" styleClass="slds-p-bottom_x-large">
+
+            <c:UTIL_PageMessages />
+
+            <div class="slds-card__header slds-grid">
+                <div class="slds-no-flex">
+                    <apex:commandButton value="Delete Data Imports"
+                        onclick="return confirm('Do you want to proceed deleting Data Imports?');"
```

Comment:
The text for this should be moved into a label. 

@smarovich What text would you like for this? This is a confirmation for the Delete All Data Imports button on the NPSP Data Imports tab.

![screen shot 2018-11-30 at 2 35 37 pm](https://user-images.githubusercontent.com/2000501/49318401-49d4c300-f4ad-11e8-8cff-1f9d89dd3540.png)

Reply:
@lmeerkatz will move this message momentarily into the label and change it when Sarah confirms.